appcelerator-archive/amp
# Add micro.js example with build

adds to https://github.com/appcelerator/amp/issues/385

Adds an example of building and adding an image to the local registry
Adds a single function style application

To test:
go to `examples/micro`
run `./deploy`
check with `curl http://www.micro.localhost.tv/test` or browser
